### ISSUE TITLE
API - Modify simple user to admin of the poll / Delegate admin on a poll

### ISSUE THREAD

**Issue by Youcef-Volya** [nextcloud/polls#3684]
### ⚠️ This issue respects the following points: ⚠️

- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Describe the goal you'd like to achieve

As a **poll owner**, I want to **be able to add/modify/remove a user to be a delegate admin of the poll, using API request**, because **it's like that on the web interface and a route exist on the webapp**.

**Comment by dartcafe** [comment]
Yep. This is missing. Completing the API and get it to non-draft state is is scheduled after the vue3 migration.
